### PR TITLE
en,zh: add v1.6 GA release notes and whats new

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.6 (TiDB Operator 1.6 versions)
- [ ] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Support setting `maxSkew`, `minDomains` and `nodeAffinityPolicy` in `topologySpreadConstraints` for components of a TidbCluster ([#5617](https://github.com/pingcap/tidb-operator/pull/5617), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support set extra command line arguments for TiDB component ([#5624](https://github.com/pingcap/tidb-operator/pull/5624), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support setting `nodeSelector` for the TidbInitializer component ([#5594](https://github.com/pingcap/tidb-operator/pull/5594), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## Improvements
+
+- Support automatically set location labels for TiProxy ([#5649](https://github.com/pingcap/tidb-operator/pull/5649), [@djshow832](https://github.com/djshow832))
+- Support retrying to evict leader when rolling upgrade TiKV ([#5613](https://github.com/pingcap/tidb-operator/pull/5613), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support setting `advertise-addr` command line argument for TiProxy component ([#5608](https://github.com/pingcap/tidb-operator/pull/5608), [@djshow832](https://github.com/djshow832))
```
Comment (Oreoxmt) [inline en/releases/release-1.6.0.md:22]:
```suggestion
- Support automatically setting location labels for TiProxy ([#5649](https://github.com/pingcap/tidb-operator/pull/5649), [@djshow832](https://github.com/djshow832))
- Support retrying leader eviction during TiKV rolling restart ([#5613](https://github.com/pingcap/tidb-operator/pull/5613), [@csuzhangxc](https://github.com/csuzhangxc))
- Support setting the `advertise-addr` command-line argument for the TiProxy component ([#5608](https://github.com/pingcap/tidb-operator/pull/5608), [@djshow832](https://github.com/djshow832))
```

File: zh/releases/release-1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- 支持为 TiDB 集群各组件的 `topologySpreadConstraints` 设置 `maxSkew`、`minDomains` 与 `nodeAffinityPolicy` ([#5617](https://github.com/pingcap/tidb-operator/pull/5617), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持为 TiDB 组件设置额外的命令行参数 ([#5624](https://github.com/pingcap/tidb-operator/pull/5624), [@csuzhangxc](https://github.com/csuzhangxc))
+- 支持为 TidbInitializer 组件设置 `nodeSelector` ([#5594](https://github.com/pingcap/tidb-operator/pull/5594), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## 优化提升
+
+- 支持自动为 TiProxy 设置 location labels ([#5649](https://github.com/pingcap/tidb-operator/pull/5649), [@djshow832](https://github.com/djshow832))
+- 支持在滚动重启 TiKV 时，对 evict leader 执行重试 ([#5613](https://github.com/pingcap/tidb-operator/pull/5613), [@csuzhangxc](https://github.com/csuzhangxc))
+- 为 TiProxy 组件设置 `advertise-addr` 参数 ([#5608](https://github.com/pingcap/tidb-operator/pull/5608), [@djshow832](https://github.com/djshow832))
```
Comment (Oreoxmt) [inline zh/releases/release-1.6.0.md:22]:
```suggestion
- 支持为 TiProxy 组件设置 `advertise-addr` 参数 ([#5608](https://github.com/pingcap/tidb-operator/pull/5608), [@djshow832](https://github.com/djshow832))
```

File: en/whats-new-in-v1.6.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Support deploying PD v8.0.0 and later versions in [microservice mode](https://docs.pingcap.com/tidb/dev/pd-microservices) (experimental).
+- Support scaling out or in TiDB components in parallel.
+
+## Usability
+
+- Support automatically set location labels for TiProxy.
+- Support setting `maxSkew`, `minDomains` and `nodeAffinityPolicy` in `topologySpreadConstraints` for components of a TidbCluster.
+- Support setting `startupProbe` for TiDB components.
+- Support set extra command line arguments for TiDB component.
```
Comment (Oreoxmt) [inline en/whats-new-in-v1.6.md:25]:
```suggestion
- Support automatically setting location labels for TiProxy.
- Support setting `maxSkew`, `minDomains`, and `nodeAffinityPolicy` in `topologySpreadConstraints` for components of a TiDB cluster.
- Support setting `startupProbe` for TiDB components.
- Support setting additional command-line arguments for TiDB components.
```

File: zh/whats-new-in-v1.6.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Summary: 了解 TiDB Operator 1.6.0 版本引入的新特性。
+---
+
+# TiDB Operator v1.6 新特性
+
+TiDB Operator v1.6 引入了以下关键特性，从扩展性、易用性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
+
+## 兼容性改动
+
+- 升级 Kubernetes 依赖库至 v1.28 版本，建议不再部署 tidb-schedule 组件。
```
Comment (Oreoxmt) [inline zh/whats-new-in-v1.6.md:12]:
```suggestion
- 升级 Kubernetes 依赖库至 v1.28 版本，建议不再部署 tidb-scheduler 组件。
```

**Review by Oreoxmt — APPROVED** [review]

File: zh/whats-new-in-v1.6.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Summary: 了解 TiDB Operator 1.6.0 版本引入的新特性。
+---
+
+# TiDB Operator v1.6 新特性
+
+TiDB Operator v1.6 引入了以下关键特性，从扩展性、易用性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
+
+## 兼容性改动
+
+- 升级 Kubernetes 依赖库至 v1.28 版本，建议不再部署 tidb-scheduler 组件。
```
Comment (qiancai) [inline zh/whats-new-in-v1.6.md:12]:
```suggestion
- 升级 Kubernetes 依赖库至 v1.28 版本，建议不再部署 `tidb-scheduler` 组件。
```

File: zh/whats-new-in-v1.6.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- 通过 Helm Chart 部署时支持设置 tidb-controller-manager 用于 leader 选举的 lock resource，默认值为 `.Values.controllerManager.leaderResourceLock: leases`。当从 v1.6 之前的版本升级到 v1.6.0-beta.1 及之后的版本时，推荐先设置 `.Values.controllerManager.leaderResourceLock: endpointsleases` 并待新的 tidb-controller-manager 正常运行后再设置为 `.Values.controllerManager.leaderResourceLock: leases` 以更新部署。
```
Comment (qiancai) [inline zh/whats-new-in-v1.6.md:13]:
```suggestion
- 当通过 Helm Chart 部署时，支持设置 `tidb-controller-manager` 用于 leader 选举的 lock resource，默认值为 `.Values.controllerManager.leaderResourceLock: leases`。当从之前的版本升级到 v1.6.0-beta.1 或之后的版本时，推荐先设置 `.Values.controllerManager.leaderResourceLock: endpointsleases`，等待新的 `tidb-controller-manager` 正常运行后再设置 `.Values.controllerManager.leaderResourceLock: leases` 以更新部署。
```

File: en/whats-new-in-v1.6.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# What's New in TiDB Operator v1.6
+
+TiDB Operator 1.6 introduces the following key features, which helps you manage TiDB clusters and the tools more easily in terms of extensibility and usability.
+
+## Compatibility changes
+
+- Upgrade Kubernetes dependency to v1.28, and it is not recommended to deploy tidb-scheduler.
+- When deploying using Helm chart, support setting lock resource used by tidb-controller-manager for leader election, with the default value of `.Values.controllerManager.leaderResourceLock: leases`. When upgrading from versions before v1.6 to v1.6.0-beta.1 and later versions, it is recommended to first set `.Values.controllerManager.leaderResourceLock: endpointsleases` and wait for the new tidb-controller-manager to run normally before setting it to `.Values.controllerManager.leaderResourceLock: leases` to update the deployment.
```
Comment (qiancai) [inline en/whats-new-in-v1.6.md:13]:
```suggestion
- When deploying using Helm chart, support setting lock resource used by `tidb-controller-manager` for leader election, with the default value of `.Values.controllerManager.leaderResourceLock: leases`. When upgrading TiDB Operator to v1.6.0-beta.1 or a later version, it is recommended to first set `.Values.controllerManager.leaderResourceLock: endpointsleases` and wait for the new `tidb-controller-manager` to run normally before setting it to `.Values.controllerManager.leaderResourceLock: leases` to update the deployment.
```

File: en/whats-new-in-v1.6.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+## Usability
+
+- Support automatically setting location labels for TiProxy.
+- Support setting `maxSkew`, `minDomains`, and `nodeAffinityPolicy` in `topologySpreadConstraints` for components of a TiDB cluster.
+- Support setting `startupProbe` for TiDB components.
+- Support setting additional command-line arguments for TiDB components.
+- Support setting `livenessProbe` and `readinessProbe` for the Discovery component.
+- Support setting `nodeSelector` for the TidbInitializer component.
+- Support for TiFlash to directly mount ConfigMap without relying on an InitContainer to process configuration files.
```
Comment (qiancai) [inline en/whats-new-in-v1.6.md:28]:
```suggestion
- Enable TiFlash to directly mount ConfigMap without relying on an InitContainer to process configuration files.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**